coffea/processor/executor.py

```diff
@@ -1073,157 +817,450 @@ def run_uproot_job(
             measurements. Currently only supported is 'walltime' (default 60s).
     """
 
-    import warnings
+    executor: ExecutorBase
+    pre_executor: Optional[ExecutorBase] = None
+    chunksize: int = 100000
+    maxchunks: Optional[int] = None
+    metadata_cache: Optional[MutableMapping] = None
+    dynamic_chunksize: bool = False
+    dynamic_chunksize_targets: Dict = field(default_factory=dict)
+    skipbadfiles: bool = False
+    xrootdtimeout: Optional[int] = None
+    align_clusters: bool = False
+    savemetrics: bool = False
+    mmap: bool = False
+    schema: schemas.BaseSchema = schemas.BaseSchema
```

Comment:
`Optional[schemas.BaseSchema]` (at least until we deprecate LazyDataFrame)

Reply:
done